src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -301,15 +303,23 @@ function OptionRowLHN({reportID, isFocused = false, onSelectRow = () => {}, opti
                                                     </Tooltip>
                                                 )}
                                             </View>
-                                            {optionItem.alternateText ? (
+                                            {!!optionItem.alternateText && !alternateTextContainsCustomEmoji && (
                                                 <Text
                                                     style={alternateTextStyle}
                                                     numberOfLines={1}
                                                     accessibilityLabel={translate('accessibilityHints.lastChatMessagePreview')}
                                                 >
                                                     {Parser.htmlToText(optionItem.alternateText)}
```

Comment:
Instead of copying this entire block, can we just put the changes for `!alternateTextContainsCustomEmoji` and `alternateTextContainsCustomEmoji`  inside the text component? 

a bit more DRY